src/RestWrite.js

```diff
@@ -1197,7 +1197,11 @@ RestWrite.prototype._updateResponseWithData = function(response, data) {
     const dataValue = data[fieldName];
     const responseValue = response[fieldName];
 
-    response[fieldName] = responseValue || dataValue;
+    if(responseValue || responseValue === 0) {
```

Comment:
We should use response.hasOwnProperty(fieldName) and I believe it should fit the bill :)

Reply:
Ah nice thinking!